src/contracts/Chef.sol

```diff
@@ -209,16 +51,18 @@ contract Chef is AccessControl {
         _;
     }
 
-    //------------------------------------------------------Structs
-
     event Harvest(address harvester, uint256 agIndex, uint256 amount);
     event Withdraw(address withdrawer, uint256 agIndex, uint256 amount);
 
-    struct AllocationGroup {
-        address groupAddress;
-        uint256 shares;
-        bool autodistribute;
-        uint256 rewardDebt; // keeps track of how much the user is owed or has been credited already
-        uint256 claimable;
-    }
 }
+
+// DO WE ADD THESE TO MAIN CHEF OR PER TYPE OF CHEF
```

Comment:
can ya start the first line `TODO` so it's easy to find?  also recommend the Todo extension for vscode https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree